cluster/cluster_SUITE.erl

```diff
@@ -188,6 +189,35 @@ acks(Config) ->
 
     Config.
 
+concurrent_fetch(_Config) ->
+    Topic = vg_test_utils:create_random_name(<<"cluster_concurrent_fetch">>),
+    RandomRecords = [crypto:strong_rand_bytes(60), crypto:strong_rand_bytes(60),
```

Comment:
Yea, doesn't apply here. The random messages are constructed once and then used by all the different processes.